set.mm

```diff
@@ -43875,16 +43893,16 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
 
   ${
     tpnz.1 $e |- A e. _V $.
-    $( A triplet containing a set is not empty.  (Contributed by NM,
```

Comment:
I see you removed the _unordered_ mention for pairs, why add it for triples?
Can't we, similarly, consider triple means unordered by default ?

Reply:
Now I notice @icecream17's remark: this is basically the same...

Reply:
"Pairs", are "by default" unordered, though maybe the default is not that standard, and when the context is not sufficient, it may be more clear to write explicitly (even if redundantly) "unordered pair".  Maybe I removed it in this PR at places where it should have remained for clarity ?

On the other hand, tuples (hence all terms ending in -uple / -iple) or by default ordered, and here the default is really standard (https://en.wikipedia.org/wiki/Tuple).

Reply:
Ok, I see!

Reply:
I just added a clarification in my latest commit (and hopefully last). Sorry to ask you if you can approve it, @tirix !